File: Models/Item.cs
namespace TradingPost.Models
{
    public class Item
    {
        public int Id { get; set; }
        public int SellerId { get; set; }
        public string Description { get; set; }
        public int Price { get; set; }
        public bool Trade { get; set; }
        public int UserProfileId { get; set; }
        public byte[] Picture { get; set; }
    }
}

File: Client/trading-post/src/Components/Items/Item.js
import { useEffect } from "react";
import { CardBody } from "reactstrap";
import { getItemById } from "../../Managers/ItemManager.js";
import React from "react";


export const Item = ({item}) => {

    useEffect(() => {
        getItemById(item.id).then((fetchedItem) => {

        })
    }, [item.id])
    return (
        <CardBody>
            <div>
                <strong className="item-description">
                    {item.Description}
                </strong>
                <div>
                    {item.Price}
                </div>
                <div>
                    {item.Trade}
                </div>
                <div>
                    {item.Picture}
                </div>
            </div>
        </CardBody>
    );
}